tests/unit/test_parsers_jats.py

```diff
@@ -159,5 +159,15 @@ def test_collaborations(aps_jats, aps_parsed):
     assert result == expected
 
 
+def test_references(aps_jats, aps_parsed):
+    result_refs = aps_jats.parse_references()
+    expected = aps_parsed['references']
+
+    assert len(result_refs) == len(expected)
+
+    for idx in range(len(expected)):
```

Comment:
you can simply `assert result_refs == expected`, no need to teach `pytest` what it means for two lists to be equal.